CHANGES.md

```diff
@@ -1,15 +1,23 @@
+# 1.10.0 (13 August 2021)
+
+* [+] A link to the extension was added to the left menu in Power User mode and Service Provider View.
```

Comment:
* [+] Added the extension tab to the left navigation pane in Power User and Service Provider views.